styles.css

```diff
@@ -10,6 +10,7 @@ html {
 
```

Comment:
- [ ] Nice work, but there is still a little more to do about the fonts.

Everything is important:  size, weight, line-height, font family, style.   It seems you have been misregarding the `font-weight` and the `line-height` properties for the fonts.  Please include them so the project looks like the figma.

- [ ] There is a problem with the languages boxes (HTML, CSS, javascript, etc).  Here is the solution:

![image](https://user-images.githubusercontent.com/57421823/156412093-67687ecc-bfcd-45b2-8c0d-03ce30060a34.png)

Just remove that `width: 20%` property and it should work! (tested of firefox and chrome)💪 

- [ ] These commas are not featured on the Figma, therefore, please remove them:

![image](https://user-images.githubusercontent.com/57421823/156413172-a921a881-e117-42df-8d1d-ef5b7300075d.png)

If you implement these changes, your project will look like the Figma! 💪

Reply:
Thanks for addressing those issues, and I appreciate your feedback it was very clear.
I've made the following changes :
- Removed the width property from the list items.
- Removed commas by using the JS join method